docker/container.go

```diff
@@ -190,14 +190,26 @@ func (c *Container) Delete() error {
 		return err
 	}
 
-	if info.State.Running {
-		err := c.client.StopContainer(container.ID, c.service.context.Timeout)
-		if err != nil {
-			return err
-		}
+	if !info.State.Running {
+		return c.client.RemoveContainer(dockerclient.RemoveContainerOptions{ID: container.ID, Force: true, RemoveVolumes: c.service.context.Volume})
```

Comment:
I don't think we want `Force: true`. We're already checking if it's stopped, so we shouldn't need to force it.